web_app/startup.sh

```diff
@@ -1,16 +1,12 @@
-
 # !/bin/bash
 
 # Run Backend 
-echo "Running app.py..."
-python3 app.py &
```

Comment:
Is there a reason to removing the control operator `&`? 

If you fix app.py, without the operator the frontend doesn't run.

Reply:
Yeah that was the typo I was talking about